server/plugins/error-pages.js

```diff
@@ -7,29 +8,7 @@ module.exports = {
     name: 'error-pages',
     register: (server, options) => {
       server.ext('onPreResponse', (request, h) => {
-        const response = request.response
-
-        if (response.isBoom) {
-          // An error was raised during
-          // processing the request
-          const statusCode = response.output.statusCode
-
-          // In the event of 404
-          // return the `404` view
-          if (statusCode === 404) {
-            return h.view('404').code(statusCode)
-          }
-
-          request.log('error', {
-            statusCode: statusCode,
-            data: response.data,
-            message: response.message
-          })
-
-          // The return the `500` view
-          return h.view('500').code(statusCode)
-        }
-        return h.continue
+        return request.response.isBoom ? processErrorResponse(request) : h.continue

```

Comment:
Replaced the invalid code with the implementation from the base project PR that handes 400 errors correctly